frontend/src/scenes/feature-flags/featureFlagsLogic.ts

```diff
@@ -40,7 +40,7 @@ export const featureFlagsLogic = kea<featureFlagsLogicType>({
         featureFlags: {
             __default: [] as FeatureFlagType[],
             loadFeatureFlags: async () => {
-                const response = await api.get(`api/projects/${values.currentTeamId}/feature_flags`)
+                const response = await api.get(`api/projects/${values.currentTeamId}/feature_flags/?limit=300`)
```

Comment:
bumped it to 300 because it seems more reasonable, and anything more than that warrants a revisit.